src/generate/register.rs

```diff
@@ -246,7 +239,6 @@ pub fn fields(
         let fty = width.to_ty()?;
         let evs = &f.enumerated_values;
         let quotedfield = String::from("`") + &f.name + "`";
```

Comment:
Maybe this `let quotedfield` line should be moved too. @therealprof could you comment on this?

Reply:
No preference. I planned to do something with it but that plan is already gone out the window with this PR anyway. 😅

Reply:
Ok! @burrbull, could you move it then to line 296?